.github/ISSUE_TEMPLATE/NextJS Bug.md

```diff
@@ -0,0 +1,79 @@
+---
+name: Bug report
+about: Create a bug report for Next js 
+title: ""
+labels: 'bug'
+assignees: ''
+
+---
+
+<!-- 
+  Please note this template is only to report bugs found in Frontend of Autodl.
+  FontEndApp/autodl-platform uses Nextjs in frontend, follow the guide (https://github.com/Auto-DL/Auto-DL/tree/main/FrontEndApp/autodl-platform)
+  to run the code in your localhost before creating the bug report.  
+-->
+
+Thanks for taking the time to file a bug report! Please fill the details as completely as possible. 
+
+```
+Report bugs which you have discovered in FrontEndApp/autodl-platform
+```
+
+Before reporting the bug check if the bug already exits [here](https://github.com/Auto-DL/Auto-DL/issues)
+
+
+## Bug Report
+
+<!-- Please try to summarize the bug in a few lines -->
+
+### Current Behavior
+<!-- A clear and concise description of the behavior.  If possible please provide a minimal code sample or steps to replicate the issue -->
+
+
+### Steps to Reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+
+### Expected behavior/code
+<!-- A clear and concise description of what you expected to happen (or code). -->
+
+
+### ⚙️ Environment
+
+- `OS`: [e.g. Ubuntu 18.04, OSX 10.13.4, Windows 10]
+- `Node`: [ e.g. 14.18.1 ]
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. chrome, safari, firefox]
+ - Version [e.g. 22]
+
```

Comment:
comment this out.

Reply:
Instead of commenting the entire thing out... we can just comment out the lines 49 to 58 which could serve as an example rather and 46-47 being compulsory fields. Just for better understanding

Reply:
That could be done but don't keep the node version compulsory. The user might be just browsing and might not know what node version he is using.